docs/source/harvesters/scanr.rst

```diff
@@ -0,0 +1,25 @@
+Harvesters : ScanR
+=====================================
+
+Data source
+-----------
+
+SoVisu+ ScanR Harvester fetches data from the latest ScanR API.
+
+.. note:: To operate the ScanR Harvester, identifiers and credentials from the non-public ScanR API are needed. These can be procured by reaching out to the ScanR team and should be integrated into the harvester's environment variables. The harvester will be updated to use the public ScanR API as soon as it is available.
```

Comment:
In my opinion it's not a "non-public API", it's the next version, still not in production.